src/clj/puppetlabs/puppetserver/cli/gem.clj

```diff
@@ -1,10 +1,12 @@
 (ns puppetlabs.puppetserver.cli.gem
   (:require [puppetlabs.puppetserver.cli.subcommand :as cli]
-            [puppetlabs.services.jruby.jruby-puppet-core :as jruby-core]))
+            [puppetlabs.services.jruby.jruby-puppet-core :as jruby-puppet-core]
+            [puppetlabs.services.jruby.jruby-core :as jruby-core]))
 
 (defn gem-run!
   [config args]
-  (jruby-core/cli-run! config "gem" args))
+  (let [jruby-config (jruby-core/initialize-config (jruby-puppet-core/extract-jruby-config (:jruby-puppet config)))]
```

Comment:
This should now be able to be changed to use the new `initialize-configs` function.